inc/media-modal.php

```diff
@@ -106,7 +106,12 @@ public function save_fields( $post, $attachment ) {
 	 * @return bool
 	 */
 	public function is_in_normal_mode( $show, $meta_box ) {
-		return $show && ! $this->is_in_modal( $meta_box );
+		global $pagenow;
+		if ( $pagenow != 'post.php' ) {
```

Comment:
Điều kiện này nghĩa là gì em nhỉ? Có thể viết tường minh ra, hoặc đặt hẳn 1 method để giải thích rõ hơn ko? A nghĩ cũng nên dùng hàm `get_current_screen` sẽ hay hơn check qua `$pagenow` (nhớ kiểm tra `is_admin`)

Reply:
- Ở file media-modal.php:  hàm `rwmb_show` sẽ trả về `true` nếu `function is_in_normal_mode` trả về `true`
Mà function này lại check nếu ko phải modal thì mới trả về true, vậy là sai !
- Hàm này nếu đổi từ `$show && ! $this->is_in_modal( $meta_box )` --> `$show && $this->is_in_modal( $meta_box )` thì các file css/js sẽ đúng, nhưng ở trang detail attactment sẽ bị hiện duplicate 2 field ( field thường và field của modal )
- Do đó e check nếu ko ở trang `post.php` thì sẽ check điều kiện `$show && $this->is_in_modal( $meta_box )` để các `css/js` hoạt động !

Reply:
- Dạ e check thì hàm `get_current_screen` không hoạt động trên file này ạ !

Reply:
Hàm `is_in_normal_mode` trong file `modal-media.php` sẽ check xem có show `css/js và html` của field đó có `show` ra hay không

`public function is_in_normal_mode( $show, $meta_box ) {	
	return $show && ! $this->is_in_modal( $meta_box );
}`

Nếu dùng `media modal` thì `! $this->is_in_modal( $meta_box )` sẽ trả về `true` => `is_in_normal_mode` trả về `false` => biến `$show` trả về `false` và sẽ ko show css/js